src/main/java/com/dev/cinema/util/HashUtil.java

```diff
@@ -0,0 +1,37 @@
+package com.dev.cinema.util;
+
+import com.dev.cinema.exceptions.PasswordHashingException;
+import java.security.MessageDigest;
+import java.security.NoSuchAlgorithmException;
+import java.security.SecureRandom;
+import org.apache.log4j.Logger;
+
+public class HashUtil {
+    private static final Logger LOGGER = Logger.getLogger(HashUtil.class);
```

Comment:
```suggestion
    private static final Logger logger = Logger.getLogger(HashUtil.class);
```
https://google.github.io/styleguide/javaguide.html